### ISSUE TITLE
LMS Users should See and Interact with the Appropriate Course Group

### ISSUE THREAD

**Issue by lyzadanger** [hypothesis/product-backlog#680]
### Problem

Private groups are created for LMS courses, and course students are members of their course's private group. That way, annotations for the course are organized together. Right now, these private groups are created manually, but upcoming work will make the group creation and membership establishment automated:

https://github.com/hypothesis/product-backlog/issues/596
https://github.com/hypothesis/product-backlog/issues/595

However, it is easy to lose track of that private group in the h client interface. Currently the client will focus either the last group a user posted to, or will fall back to focusing the "Public" group. Students often start annotating, by accident, within the Public group, not realizing at all that a private group for their course even exists—it's hidden within the groups menu.

### User Stories

* As an LMS student using Hypothesis, I want my current course's group to be automatically selected for me; that way I know I'm automatically annotating in the correct group and I can get started annotating right away.
* As an LMS student using Hypothesis, I only need to be able to see and access the group for my current course; being able to see, select or act within other courses could cause me to mistakenly make annotations in the wrong place.
* As an LMS instructor using Hypothesis, I want to feel confident that my students are making annotations only within my course's group; that way, I know that my course's group contains all of the relevant student annotations.

### Acceptance Criteria

- [x] When the embedded h LMS app is launched, the appropriate course group should be 
active (selected/focused in the client)
- [x] It should not be possible for the LMS user to accidentally post annotations in the wrong course group—those groups should be inaccessible to the user
- [x] It should not be possible for an LMS user to leave any of their course groups
- [x] It should not be possible for an LMS user to create new groups

**Comment by dwhly** [comment]
Making a note that this is related (or possibly identical) to adding the group facet for bouncer links here. (https://github.com/hypothesis/client/issues/488).  There are other partners waiting on this capability, so ensuring that passing the parameter via bouncer links is supported would be a win.

**Comment by lyzadanger** [comment]
Please note the description and title has been edited to capture a few nuances. This card will now serve as an epic for several sub-issues.

**Comment by seanh** [comment]
@lyzadanger I wonder if we can close this? I believe the acceptance criteria are done. The _leave group_ link is still shown, but we have a separate issue for hiding that link / hiding the groups menu entirely, and even with the _leave group_ link still present you can only temporarily leave the group (it just adds you back next time you launch an assignment). Apart from that I think the other criteria are done?

**Comment by lyzadanger** [comment]
@seanh I'm going to leave this open until we execute on #868

**Comment by lyzadanger** [comment]
With the merging of https://github.com/hypothesis/product-backlog/issues/868, we have completed this epic!
